po/fr.po

```diff
@@ -16892,408 +16843,420 @@ msgstr "Changer le répertoire de travail"
 
 #: /tmp/fish/implicit/share/completions/find.fish:44
 msgid "File name matches regex"
-msgstr ""
+msgstr "Le nom du fichier correspond à l’expression régulière"
 
 #: /tmp/fish/implicit/share/completions/find.fish:45
 msgid "File refers to the same inode as specified file"
-msgstr ""
+msgstr "Le fichier se réfère au même i-nœud que le fichier spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:46
 msgid "File uses specified units of space"
-msgstr ""
+msgstr "Le fichier occupe la taille spécifiée"
 
 #: /tmp/fish/implicit/share/completions/find.fish:47
-#, fuzzy
 msgid "Always true"
-msgstr "Toujours"
+msgstr "Toujours vrai"
 
 #: /tmp/fish/implicit/share/completions/find.fish:48
 msgid "File is of specified type"
-msgstr ""
+msgstr "Le fichier est du type spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:49
 msgid "File's owner has specified numeric user ID"
-msgstr ""
+msgstr "UID de l’utilisateur propriétaire du fichier"
 
 #: /tmp/fish/implicit/share/completions/find.fish:50
 msgid ""
 "File was last accessed specified number of days after its status was last "
 "changed"
 msgstr ""
+"Fichier dont le dernier accès date de n jours après sa date de dernière "
+"modification"
 
 #: /tmp/fish/implicit/share/completions/find.fish:51
 msgid "File's owner"
-msgstr ""
+msgstr "Utilisateur propriétaire du fichier"
 
 #: /tmp/fish/implicit/share/completions/find.fish:52
+#, fuzzy
 msgid ""
 "Check type of file - in case of symlink, check the file that is not checked "
 "by -type"
 msgstr ""
+"Vérifier de type du fichier — s’il est un lien symbolique, vérifier le "
+"fichier qui n’est pas vérifié par -type"
 
 #: /tmp/fish/implicit/share/completions/find.fish:53
 msgid "File's security context matches specified pattern"
-msgstr ""
+msgstr "Le contexte SELinux du fichier correspond au motif spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:54
-#, fuzzy
 msgid "Delete selected files"
-msgstr "Supprimer les fichiers obsolètes"
+msgstr "Supprimer les fichiers trouvés"
 
 #: /tmp/fish/implicit/share/completions/find.fish:55
-#, fuzzy
 msgid "Execute specified command for each located file"
-msgstr "Exécuter la commande si la précédente a échoué"
+msgstr "Exécuter la commande spécifiée pour chaque fichier trouvé"
 
 #: /tmp/fish/implicit/share/completions/find.fish:56
 msgid "Execute specified command for each located file, in the files directory"
 msgstr ""
+"Exécuter la commande spécifiée pour chaque fichier trouvé, dans le dossier "
+"contenant le fichier"
 
 #: /tmp/fish/implicit/share/completions/find.fish:57
 msgid "List file in ls -dils format, write to specified file"
 msgstr ""
+"Enregistrer, dans le fichier spécifié, les chemins des fichiers au format ls "
+"-dils"
 
 #: /tmp/fish/implicit/share/completions/find.fish:58
-#, fuzzy
 msgid "Print full file names into specified file"
-msgstr "Spécifier les noms de fichier à filtrer"
+msgstr "Afficher les chemins absolus des fichiers séparés par \\0"
```

Comment:
Incorrect translation, should be “Enregistrer, dans le fichier spécifié, les chemins absolus des fichiers” (compare with translations below).

Reply:
Corrected.